### PR TITLE
feat(connect-deeplink): deeplinkUrl setting (for app linking)

### PR DESCRIPTION
**Pull request by martykan**
## Description

Add `deeplinkUrl` setting to allow changing the URL to Suite. 
Add intent filters for opening the app from https://dev.suite.sldev.cz/connect/.../deeplink/
Update example app URL for testing
Add [fallback page](https://dev.suite.sldev.cz/connect/feat/native-connect-popup-deeplink-url/deeplink/) for deeplinks (telling user to download app)

### CODE REVIEW

File: packages/connect-deeplink/src/index.ts [hunk 1]
```diff
@@ -174,7 +175,7 @@ export class TrezorConnectDeeplink implements ConnectFactoryDependencies {
     }
 
     private buildUrl(method: string, params: any, callback: string) {
-        return `${this.schema}://connect?method=${method}&params=${encodeURIComponent(
+        return `${this._settings.deeplinkUrl}?method=${method}&params=${encodeURIComponent(
```
Comment (karliatto) [inline packages/connect-deeplink/src/index.ts:170]: I am having here some changes related to this https://github.com/trezor/trezor-suite/pull/14490/files#diff-161cfff841ef61661da9b9f9b576e2f9d4d96db4dbc697b6202506404d5acdcdR4

But we can adjust once we are ready to merge it.

File: suite-native/app/app.config.ts [hunk 2]
```diff
@@ -169,6 +169,29 @@ export default ({ config }: ConfigContext): ExpoConfig => {
                 monochromeImage: './assets/appIcon_android.png',
                 ...appIconAndroid,
             },
+            intentFilters:
```
Comment (karliatto) [inline suite-native/app/app.config.ts:171]: If I understand well this configuration is only for Android, right?

Comment (martykan) [reply suite-native/app/app.config.ts:171]: Yes this PR only supports Android, I would leave iOS for later, after the testing phase

**Review by karliatto — APPROVED** [review]
Looks good.

**Pull request merged**